chunker/scripts/runBenchmarkTest.sh

```diff
@@ -17,7 +17,7 @@ for JAR in `ls $LIB`; do
 done
 
 
-TESTDATA="src/main/resources/test.txt"
+TESTDATA="/shared/corpora/corporaWeb/written/eng/chunking/conll2000distributions/test.txt"
```

Comment:
What do you mean by "not used anyhow"? 
Why don't we use only the links to [the files that we already have in the repo](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/tree/master/chunker/src/test/resources)?